.buildkite/bisect/config.yml

```diff
@@ -0,0 +1,22 @@
+name: ray-bisect
+artifacts_bucket: ray-ci-artifact-branch-public
+ci_temp: s3://ray-ci-artifact-branch-public/ci-temp/
+ci_work_repo: 029272617770.dkr.ecr.us-west-2.amazonaws.com/rayproject/citemp
+forge_prefix: cr.ray.io/rayproject/
+builder_queues:
+  builder: builder_queue_branch
+runner_queues:
+  default: runner_queue_small_branch
+  macos: macos-branch
+buildkite_dirs:
+  - .buildkite/bisect
+env:
+  BUILDKITE_BAZEL_CACHE_URL: https://bazel-cache-dev.s3.us-west-2.amazonaws.com
+  RAYCI_SKIP_UPLOAD: "true"
+  # the following will be replaced by the actual values in the bisect pipeline
+  RAYCI_BISECT_TEST_NAME: __RAYCI_BISECT_TEST_NAME__
+  RAYCI_BISECT_PASSING_COMMIT: __RAYCI_BISECT_PASSING_COMMIT__
+  RAYCI_BISECT_FAILING_COMMIT: __RAYCI_BISECT_FAILING_COMMIT__
+  RAYCI_BISECT_TEST_TYPE: __RAYCI_BISECT_TEST_TYPE__
```

Comment:
> the following will be replaced by the actual values in the bisect pipeline

how? and why? why do you need to define them here?

Reply:
yess, check out the bisect pipeline definition https://buildkite.com/ray-project/bisect/settings/steps

the `test_name`, etc. are provided as input through buildkite UI. I need a way to generate this config file based on that input in the init function. Not sure what is the best way to do so but this is what I can come up with. If there is a way to pass a list of environment variable that could work too.